sysdig/resource_sysdig_secure_cloud_auth_account.go

```diff
@@ -21,6 +19,59 @@ import (
 	cloudauth "github.com/draios/terraform-provider-sysdig/sysdig/internal/client/v2/cloudauth/go"
 )
 
+/*
+declare common schemas used across resources here
+*/
+var (
+	accountComponents = &schema.Resource{
```

Comment:
nit: should this be singular?

Reply:
actually yes, singular makes sense for SchemaComponent usage in both account and component resources. I'll fix this when I touch this part of code again.